utility/src/main/java/org/oppia/util/parser/CustomBulletSpan.kt

```diff
@@ -36,6 +32,10 @@ class CustomBulletSpan(
 
   private var mBulletPath: Path? = null
```

Comment:
Nit: should be ``bulletPath`` since we don't use Android's naming style for class members.

Reply:
Done.